roles/server-base/vars/main/vars.yml

```diff
@@ -0,0 +1,4 @@
+site: influxdb-proxy
+site_name: influxdb proxy
+server_name: influxdb.yunity.org
+redirects: []
```

Comment:
How does this work when using server-base for other playbooks which set these variables seperately? If the playbook-defined vars (e.g. `playbooks/karrot-world/setup.playbook.yml`) correctly overwrite them, maybe add a note here to explain? Otherwise it's a bit confusing what they are kind of reused for a different purpose.

Reply:
Another idea would be to just treat influxdb-proxy as a seperate site with it's own folder and playbooks...